### PR TITLE
Feature/transaction errors

### PR DESCRIPTION
**Pull request by dabasov**
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zwalletcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master BEFORE merging this PR

Associated PRs (Link as appropriate):
- [0chain](https://github.com/0chain/0chain/pull/930)
- [gosdk](https://github.com/0chain/gosdk/pull/361)
- [zboxcli](https://github.com/0chain/zboxcli/pull/158)
- [system_tests](https://github.com/0chain/system_test/pull/167)
...

### CODE REVIEW

File: cmd/lock.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ var lockcmd = &cobra.Command{
 				ExitWithError(err.Error())
 			}
 			if statusBar.success {
-				fmt.Printf("\nTokens (%f) locked successfully\n", token)
+				fmt.Printf("\nTokens (%f) locked successfully\nHash: %v", token, txn.GetTransactionHash())
```
Comment (stewartie4) [inline cmd/lock.go]: this is great!
I believe send zcn should also return a hash

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | feature/transaction-errors  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | feature/transaction-errors  |
| zwalletcli  | feature/transaction-errors  |
| 0chain  | feature/transaction-errors  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Review by cnlangzi — APPROVED** [review]
LGTM

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | feature/transaction-errors  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | feature/transaction-errors  |
| zwalletcli  | feature/transaction-errors  |
| 0chain  | feature/transaction-errors  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**